packages/datetime/src/datePicker.tsx

```diff
@@ -55,6 +55,18 @@ export interface IDatePickerProps extends IDatePickerBaseProps, IProps {
      */
     showActionsBar?: boolean;
 
+    /**
+     * Today action bar button title.
+     * @default "Today"
+     */
+    todayButtonText?: string;
+
+    /**
+     * Reset action bar button title.
```

Comment:
weird to say `title` here and `Text` in the prop name. consistency please. same for the prop above.

> Text for the reset button in the action bar.